### ISSUE TITLE
style suggestion: JAX tutorial 5

### ISSUE THREAD

**Issue by murphyk** [phlippe/uvadlc_notebooks#86]
In https://uvadlc-notebooks.readthedocs.io/en/latest/tutorial_notebooks/JAX/tutorial5/Inception_ResNet_DenseNet.html,
replace
```
# Opens tensorboard in notebook. Adjust the path to your CHECKPOINT_PATH!
%tensorboard --logdir saved_models/tutorial5_jax/tensorboards/GoogleNet/
```
with
```
%tensorboard --logdir $CHECKPOINT_PATH/tensorboards/GoogleNet/
```

**Comment by murphyk** [comment]
Actually, running `%tensorboard --logdir $CHECKPOINT_PATH/tensorboards/ResNet/` later on fails in colab, with the error `Port 6006 is in use by another program`. So maybe it is better to just use
```
%tensorboard --logdir $CHECKPOINT_PATH/tensorboards/
```
and to tell the user to toggle which results to show.

**Comment by phlippe** [comment]
Good idea! To prevent the overlap of ports, I added the `--port` flag to the later tensorboards. This way you can still see the individual tensorboards. :)
Implemented in 6e642afb844f6cee89f59441df831b144e2e6ab8.
